internal/server/smb/server.go

```diff
@@ -45,6 +45,22 @@ func (s *Server) NewSmbGlobalMapping(context context.Context, request *internal.
 		isMapped = false
 	}
 
+	valid, err := s.fsServer.PathValid(context, remotePath)
```

Comment:
Another option is let the caller to perform this logic. csi-proxy just provide all APIs needed such as PathValid.

Reply:
I think we could expose `PathValid` interface in filesystem, and for this issue, let's fix it first.